setup.py

```diff
@@ -18,7 +18,7 @@
     install_requires=[
         'biopython',
         'colour',
-        'dash>=0.37.0',
+        'dash>=0.40.0',
         'dash-html-components',
```

Comment:
Can we remove this dependency, then? Just like you did in `requirements.txt`.

Reply:
@mkcor Done in 213d238 :)